bin/setup.sh

```diff
@@ -12,12 +12,12 @@ function checkPublishable() {
         echo ""
         echo "Builds in Repositories other than http4s/http4s are not published to $publishLocation"
         exit 0
-    elif [[ $TRAVIS_BRANCH != "master" || $TRAVIS_BRANCH != "release-"* ]]; then
+    elif [[ $TRAVIS_BRANCH == "master" || $TRAVIS_BRANCH == "release-"* ]]; then
```

Comment:
Couldn't this have been fixed by changing the `||` to `&&` and keeping all the failures above the fallthrough success?